ipi_tests/examples/excluded_test.txt

```diff
@@ -1,46 +1,22 @@
 # To check
-MBPOL/splitting/min
-MBPOL/splitting/instanton/32-lbfgs
-MBPOL/splitting/TS
-pes/qtip4pf/tun-splitting/min
-pes/qtip4pf/tun-splitting/instanton/32-lbfgs
-lammps/h2o-scphonons
-lammps/h2o-imf
-lammps/h2o-vscf
-lammps/h2o-extreme-rem
-lammps/isof-vapor
-pes/geop/waterbox/bfgs
-pes/zundel-systempart 
-pes/neb 
-qtip4pf/tun-splitting
-pes/qtip4pf/tun-splitting
-pes/qtip4pf/tun-splitting/TS
-lammps/ice-npt+sc 
-lammps/ice-npt-nst 
-lammps/ice-ramp-melt 
-lammps/h2o-planetary-64 
-lammps/h2o-pimd+sc+gle.4 
-lammps/isofsc-water 
-lammps/kmc-al6xxx 
-lammps/h2o-pimd+sc+mts 
-lammps/h2o-repimd 
-lammps/h2o-pimd+sc+rpc_open 
-lammps/isof-water
-lj/nst 
-tools/gleacf/h2o-wn 
-tools/gleacf/h2o-gle 
-tools/getacf 
-hswfqmc/FixWF_1Bead 
-hswfqmc/WFOpt_2Bead 
+MBPOL/splitting/instanton/32-lbfgs                                          #This is resolved with PR #123  
+pes/qtip4pf/tun-splitting/instanton/32-lbfgs                                #This is resolved with PR #123  
+lammps/h2o-planetary-64                                                     # TypeError: string argument expected, got 'bytes' 
 
 # Broken
 ffdebye/single_particle 
 lammps/h2o-constraints 
+lammps/isof-vapor                                                           # Isotope-zeta-sc is stopping the termination of the run
```

Comment:
Please after this PR gets merged, generate issues corresponding to these examples.

Reply:
yes, will do!